modules/machine-config-overview.adoc

```diff
@@ -0,0 +1,101 @@
+for details.
+
+* Although Ignition config settings can be delivered directly at {product-title}
+installation time, and are formatted in the same way that MCO delivers Ignition
+configs, MCO has no way of seeing what those original Ignition configs are. So, whenever
+possible, you should wrap Ignition config settings into a MachineConfig before
+deploying them.
+
+* When a MachineConfig managed by MCO changes outside of MCO, the Machine
+Config Daemon (MCD) sets the machine as degraded. It won't overwrite the
+offending file, however, and should continue to operate in a degraded state.
+
+* A key reason for using a MachineConfig is that it will be applied when
+you spin up new nodes in your {product-title} cluster. The machine-api-operator
+provisions a new machine and MCO configures it.
+
+== What can you change with MachineConfigs?
+The kinds of components that MCO can change include:
+
+* **config**: Create Ignition config objects (see the
+link:https://github.com/coreos/ignition/blob/master/docs/configuration-v3_1.md[Ignition configuration specification])
+to do things like modify files, systemd services, and other features on {product-title} machines, including:
+- **Configuration files**: Create or overwrite files in the `/var` or `/etc` directory.
+- **systemd units**: Create and set the status of a systemd service or add to an existing systemd service by dropping in additional settings.
+- **users and groups**: Change system user and group settings.
+- **security**: Set TLS resources for making  https connections.
+* **kernelArguments**: Add arguments to the kernel command line when {product-title} nodes boot.
+* **kernelType**: Optionally identify a non-standard kernel to use instead of the standard kernel. Use `realtime`
+to use the RT kernel.
+* **fips**: Enable link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#using-the-system-wide-cryptographic-policies_security-hardening[FIPS]
+mode. FIPS should be set at
+installation-time setting and not a post-installation procedure.
+* **extensions**: Extend {op-system} features by adding selected pre-packaged software.
+For this feature (new in {product-title} 4.6), available extensions include
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#protecting-systems-against-intrusive-usb-devices_security-hardening[usbguard] and kernel modules
+* **Custom resources (for ContainerRuntime and Kubelet)**: Outside of
+MachineConfigs, MCO manages two special custom resources for modifying
+CRI-O container runtime settings (ContainerRuntime CR) and the
+Kubelet service (Kubelet CR).
+
+The MCO is not the only facility that can change operating
+system components on {product-title} nodes. Other Operators
+can make operating system-level features as well. One example
```

Comment:
"can make operating system-level features" sounds odd to me. Is it accurate to say, "can modify" here?
Also, "Operator" should be capitalized in the link text.